File: engine/leaderboard.php
<?php
	include 'dbworks.php';

	$tojson = array();
	$query = $mysqli->query('SELECT anon_id, score, killed_count, alive FROM KillerTheGame ORDER BY score DESC');
	while($result = $query->fetch_assoc())
		$tojson[] = $result;
	if ($_GET['count'])
		echo count($tojson);
	else
		echo json_encode($tojson);
?>

File: hsespionage/leaderboard.py
import json
from hsespionage import app
from hsespionage import postgresdb
from flask import request

@app.route("/engine/leaderboard")
def leaderboard():
    leaderboard_data = postgresdb.all(
        "SELECT anon_id, score, killed_count, alive FROM hsspies_game ORDER BY score DESC",
        back_as=dict)
    if request.args.get("count"):
        return str(len(leaderboard_data))
    else:
        return json.dumps(leaderboard_data)
